OpenRA.Mods.Common/Lint/CheckInteractable.cs

```diff
@@ -42,10 +42,10 @@ void Run(Action<string> emitError, Ruleset rules, ModData modData)
 						continue;
 
 					if (HasInvalidBounds(interactable.Bounds, grid.TileSize, grid.TileScale))
-						emitError($"{nameof(interactable.Bounds)} of actor `{actorInfo.Key}` are empty or negative.");
+						emitError($"{actorInfo.Key}.{interactable.GetType().Name}.{nameof(interactable.Bounds)} are empty or negative.");
```

Comment:
Not sure if concatenating actor, trait and field all together makes anything more readable.

Reply:
Providing the trait name most definitely does